galgebra/ga.py

```diff
@@ -496,24 +496,49 @@ def coord_vec(self) -> Expr:
             raise ValueError("Ga with no coords has no coord_vec")
         return sum([coord * base for coord, base in zip(self.coords, self.basis)])
 
-    def make_grad(self, a, cmpflg=False):  # make gradient operator with respect to vector a
+    def _reciprocal_of_basis_blade(self, blade: Symbol) -> Expr:
+        r"""
+        Compute the reciprocal :math:`e^I` of a basis blade :math:`e_I`.
+
+        This is a blade :math:`e^I` such that :math:`\left<\e^Ie_J\right> = \delta_I^J`
+        (:cite:`Hestenes`, eq 3.19).
+        """
+        index = self.indexes_to_blades_dict.inverse[blade]
+        r_blade = reduce(self.wedge, [
+            self.r_basis[i] for i in index[::-1]
+        ], S.One)
+        r_blade = r_blade.simplify()
```

Comment:
I wanted to reduce `simplify()` calls to a minimal, is there any reason to call simplify here (or could there be a cache?)

Reply:
This function is cached - which is why I call simplify here.

Reply:
Oh